common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -606,6 +609,8 @@ public static String initializeDiagnosticContext(@Nullable final String requestC
         final com.microsoft.identity.common.internal.logging.RequestContext rc =
                 new com.microsoft.identity.common.internal.logging.RequestContext();
         rc.put(DiagnosticContext.CORRELATION_ID, correlationId);
+        rc.put(AuthenticationConstants.SdkPlatformFields.PRODUCT, sdkType);
+        rc.put(AuthenticationConstants.SdkPlatformFields.VERSION, sdkVersion);
```

Comment:
Can we clear the diagnostic context at the end of execution state?

Reply:
26cfa7eb0a274713270351aede7d18c256f05eef